src/NUnitTestAdapterInstall/source.extension.vsixmanifest

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Vsix xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" Version="1.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2010">
-  <Identifier Id="NUnit3TestAdapter..DD956069-EF7E-4E44-9934-33C4E4F3A793">
+  <Identifier Id="NUnit3TestAdapter">
```

Comment:
Primarily because it's not actually required and concatenating a name with a guid is ugly. :-) Admittedly, it's a more or less hidden ugliness and does no more harm than the guids in an msi.

However, after your comment I did some research. I couldn't find anyplace where MS actually _recommends_ using a guid, although they do generate one if you create a vsix project. 

In the vsix version 1 schema description, they merely say that the id must be unique. In the 2.0 schema, they are more specific:

"This attribute must be a unique ID for the package chosen by its author. The name should be qualified the same way CLR types are namespaced: Company.Product.Feature.Name. The Id attribute is limited to 100 characters."

I think feature and name come into play when adding extensions to the product.

Possible alternatives for us could be:
   NUnit.NUnit3TestAdapter
   NUnit.VisualStudio.NUnit3TestAdapter

We should take a moment to decide this, since it's inconvenient to change it later.

Charlie